lib/redirects.rb

```diff
@@ -0,0 +1,21 @@
+class Redirects
```

Comment:
Can we call this `AblyDocs::Redirects` to prevent potential conflict with the website?